pkg/monitoring/deploy.go

```diff
@@ -128,7 +128,7 @@ func DeployApp(cattleAppClient projectv3.AppInterface, projectID string, createO
 	return nil
 }
 
-func WithdrawApp(cattleAppClient projectv3.AppInterface, appLabels metav1.ListOptions) error {
+func WithdrawApp(cattleAppClient projectv3.AppInterface, appLabels metav1.ListOptions, userID string) error {
```

Comment:
I think we should change the method name. It no longer just removes the app, but updates it as well.